source/docs/software/advanced-controls/geometry/pose.rst

```diff
@@ -13,7 +13,7 @@ You can get the distance to another ``Translation2d`` object by using the ``getD
 Rotation
 --------
 
-Rotation in 2 dimensions is representated by WPILib's ``Rotation2d`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/math/geometry/Rotation2d.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc_1_1_rotation2d.html>`__). This class has an angle component, which represents the robot's rotation relative to an axis on a 2-dimensional coordinate system. Positive rotations are counterclockwise.
+Rotation in 2 dimensions is representated by WPILib's ``Rotation2d`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/math/geometry/Rotation2d.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc_1_1_rotation2d.html>`__). This class has an angle component, which represents the robot's rotation relative to an axis on a 2-dimensional coordinate system. Due to the angle being stored as its sine and cosine components, the angle will always be in the :math:`\left(-\pi, \pi\right]` range. Positive rotations are counterclockwise.
```

Comment:
Opens with a parentheses and closes with a brace

Reply:
Sorry, I'm not sure I understand?

Reply:
![E32E72E2-59EB-47A3-996B-485E7EDD18E5](https://user-images.githubusercontent.com/4885091/204051030-6aa3279e-bb83-4625-ad89-185e6d2cb9ce.jpeg)

Reply:
Never mind, it's intentional